main.js

```diff
@@ -51,6 +69,7 @@ define(function (require, exports, module) {
     var _preferences,
         _command,
         _styleTag,
+        _styleInline,
```

Comment:
This would be removed from the DOM on extension disable like the other CSS (IDK why it happens, but anyway...), but I was unable to target it for removal.